lib/event_store_http_client/events/write.rb

```diff
@@ -14,14 +14,12 @@ class Write
         dependency :settings
         dependency :client
 
-        def initialize(type, data, stream_name, version)
```

Comment:
I think that the nomenclature of "category" and "stream" are getting a bit mixed here.